CHANGES_NEXT_RELEASE

```diff
@@ -13,5 +13,6 @@
 - [BUG] Use a testing version of Zookeeper in OrionKafkaSink (#514)
 - [BUG] Always create the Hive tables if not existing yet (#401)
 - [HARDENING] Add {headers,body} structure to messages sent to Kafka (#512)
+- [FEATURE] Allow selecting the table type (by service, servicePath or destination) #540
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ae24ad3a74b0981d5e81e7a6b9136611f74708d1